modules/ROOT/pages/http/http-post-form-task.adoc

```diff
@@ -151,23 +146,17 @@ Content-Length: 32
 firstname=Aaron&lastname=Aguilar+Acevedo&yearborn=1999
 ----
 
-[TIP]
-====
-To send an HTTP request that contains form parameters, the easiest way is to use a browser extension such as Postman (Google Chrome), or the http://curl.haxx.se/[curl] command line utility. +
-If using Postman, make sure your form parameters are sent with the type `x-www-form-urlencoded`
-
-image::<missing>submit+form+aaron.png[submit+form+aaron]
-====
-
-When the request is received by the connector, it creates a Mule Message that contains a Map type payload with the following key/value pairs in it:
+When the HTTP Listener connector receives the request, it creates a Mule Message that contains a map type payload with the following key-value pairs:
```

Comment:
I am confused by "HTTP Listener connector": are we referring to HTTP Connector or some generic connector for HTTP Listener? I was confused by this earlier as well.

Reply:
Mule Message > Mule message, unless we are quoting the UI (Mule *Message* or *Mule Message*).

Reply:
"a map type payload" or a Map type payload (didn't we init. cap. types earlier?)